bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -381,4 +389,16 @@ public void handle(PullRequestBot bot, HostedCommit commit, LimitedCensusInstanc
             throw new UncheckedIOException(e);
         }
     }
+
+    private boolean verifyGroupMembership(HostedRepository targetRepo, HostUser user, PrintWriter reply) {
+        // Only check in GitLab
+        boolean hasGroupMembership = true;
+        if (!targetRepo.forge().name().equals("GitHub") && !targetRepo.forge().name().equals("Test")) {
+            hasGroupMembership = targetRepo.canPush(user);
+        }
```

Comment:
This is too much of a hack for me. There needs to be a method on the repository that each relevant implementation of the `HostedRepository` interface has a valid implementation for that answers the question we need to ask. I think that question is something like "is the user allowed to open pull requests against this repo"? For `GitHubRepository` that is probably always true, at least in our context. For `TestHostedRepository` I would assume it's always true, unless you wanted to make it false for testing purposes. In that case you could make it possible to toggle from a test. For `GitLabRepository` the method can probably delegate to `canPush`, but I would like to have that verified.

Reply:
It's quite a good idea. Will do it. Thanks!